core/cog-launcher.c

```diff
@@ -14,6 +14,7 @@
 # include "cog-gtk-utils.h"
 #endif
 
+#include <stdlib.h>
```

Comment:
This looks unrelated to the current PR. Could you please post this change in a separate PR if needed?

Reply:
Agree, it is not directly related with old versions of wayland. I will move it out.

BTW, without this it fails compilation on glibc 2.23 and gcc 5.3.

Reply:
Indeed, we still want this change applied. Let's just do it in a separate PR — you can count on an immediate :+1: for merging it :wink:

Reply:
Great. PR on the way, thanks